tuned/plugins/plugin_video.py

```diff
@@ -124,3 +199,27 @@ def _get_radeon_powersave(self, device, ignore_missing = False):
 			return "dpm-" + self._cmd.read_file(sys_files["dpm_state"]).strip()
 		else:
 			return None
+
+	@command_set("panel_power_savings", per_device=True)
+	def _set_panel_power_savings(self, value, device, sim, remove):
+		"""Set the panel_power_savings value"""
+		try:
+			value = int(value, 10)
+		except ValueError:
+			if not sim:
+				log.warn("Invalid value %s for panel_power_savings" % value)
+			return None
+		if value in range(0, 5):
+			self.target_value = value
+			if not sim:
+				return self.apply_panel_power_saving_target(device)
```

Comment:
IIUC, this will set up the signaling when handling the first device and subsequently apply the settings for *all* devices because of the loop in `upower_changed`. For the remaining devices, it's then essentially a no-op because they are already configured.

I think that's quite inelegant, but it shouldn't cause issues. Alternatively, you could remove the manual call to `upower_changed` and [not return ](https://github.com/redhat-performance/tuned/pull/601/files#diff-d503f51fb415d704c70263a4c6d4eaf2c2def141d988346452ab610d242a9da5R125)after setting up the signalling (which I guess may require an additional error check).

Reply:
I did test it on a two GPU system where both GPUs can connect to a mux for eDP and that's why it has the logic it has now; to ensure that you don't have a corner case.  Let me know if you still want it changed.

Reply:
I believe it can stay as it is now.